src/Header/index.js

```diff
@@ -32,6 +32,12 @@ export const Header = () => {
         <Button color="inherit" to="/about" component={Link}>
           About
         </Button>
+        <Button color="inherit" to="/users" component={Link}>
+          Users
+        </Button>
+        <Button color="inherit" to="/users" component={Link}>
+          User
+        </Button>
```

Comment:
This one is just duplication of previous link, they both have `to="/users"` attribute. you can delete this link